ci-operator/config/openshift/oadp-operator/.config.prowgen

```diff
@@ -1 +1,15 @@
 multi_arch: yes
+slack_reporter:
+- channel: "#oadp-cicd"
+  job_states_to_report:
+  - failure
+  - error
+  report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}*
```

Comment:
there is a preview of the message we can check to see how it will look?

Reply:
another repo with same exact template: https://github.com/kaovilai/release/blob/e6751376bf8e796fd7ac1e776dd51c2dd70be55a/ci-operator/config/openshift/image-based-install-operator/.config.prowgen#L7-L10

https://redhat.enterprise.slack.com/archives/C078MLC0SP4